### PR TITLE
Restore admin notify cooloff and remove debug logs

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Commit done after the review of the notify admin for plugin installation have changed the value of the cooldown + introduced some debug logs. This PR fixes those changes.

#### Ticket Link
N/A

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: app/notify_admin.go [hunk 1]
```diff
@@ -20,15 +20,14 @@ import (
 
 const lastTrialNotificationTimeStamp = "LAST_TRIAL_NOTIFICATION_TIMESTAMP"
 const lastUpgradeNotificationTimeStamp = "LAST_UPGRADE_NOTIFICATION_TIMESTAMP"
-const defaultNotifyAdminCoolOffDays = 0.0104166667 // this is a temp change
+const defaultNotifyAdminCoolOffDays = 14
```
Comment (marianunez) [inline app/notify_admin.go:23]: We should probably consider moving this as a configuration setting so we can override for testing in the future

**Review by marianunez — APPROVED** [review]

File: app/notify_admin.go (hunk 1 above)
Comment (JulienTant) [reply app/notify_admin.go:23]: I can create a ticket to track this, should I assign this to our team?

Comment (marianunez) [reply app/notify_admin.go:23]: You can send it over to Self-serve as the owner of Notify Admin feature

Comment (JulienTant) [reply app/notify_admin.go:23]: https://mattermost.atlassian.net/browse/MM-50425

**Review by pvev — APPROVED** [review]
LGTM!

**Pull request merged**